docs/content/developers/building-contributing.md

```diff
@@ -48,6 +50,32 @@ sudo rm /usr/local/bin/ddev
 brew link --force ddev
 ```
 
+### Linux
+
+Start by removing your current binary, for Debian flavor systems use this:
```

Comment:
I want to add that normally you don't need to remove ddev because when you put any binary in `~/.local/bin` it takes precedence.

Reply:
My approach is to use a symlink in /usr/local/bin/ddev, which points to either the system-installed ddev or the locally built one.

Reply:
We could add the symlink option as well? The reason I prefer `~/.local/bin` is that symlinks can be harder to understand.
Which commands would you use for the symlink solution?

Reply:
I use symlinking. I switch back and forth with a script I call relink_ddev.sh. Of course I have to do this all the time. My strategy is to have a symlink /usr/local/bin/ddev.link that points to the built version and then have /usr/local/bin/ddev be a symlink that points to it. But delete /usr/local/bin/ddev when using system-installed version. I use this script on Linux/WSL2 as well as macOS.

```
#!/bin/bash

set -x
set -o errexit

if command -v brew; then
  brew unlink ddev || true
fi

SUDO=""
if  [ ! -w /usr/local/bin ] || [ ! -f /usr/local/bin/ddev ] || [ ! -w /usr/local/bin/ddev ]; then
  SUDO="sudo "
fi
${SUDO} ln -sf /usr/local/bin/ddev.link /usr/local/bin/ddev
hash -r
```

Reply:
Cool method @rfay, thanks for sharing. We could include it as an alternative method, if you suggest the changes (?), I can accept the suggestion and commit it to the patch.

Reply:
Yes, you're probably right. At least, it's now documented in this issue, if anyone is interested. Resolving this thread, since it seems to be so.